_pytest/fixtures.py

```diff
@@ -1,6 +1,12 @@
 from __future__ import absolute_import, division, print_function
 import sys
 
+try:
+    from collections import OrderedDict
```

Comment:
Please check the version explicitly instead of relying on `ImportError`:

```python
if sys.version_info[:2] == (2, 6):
    from ordereddict import OrderedDict
else:
    from collections import OrderedDict
```

Using `ImportError` for that may lead to nasty surprises (see pytest-dev/pytest-mock#68 for a detailed example), plus this will make it easier for us later to find it and remove it when we drop 2.6 support.